exercises/chapter2/spago.dhall

```diff
@@ -9,7 +9,7 @@ You can edit this file as you like.
   , "foldable-traversable"
   , "integers"
   , "lists"
-  , "math"
+  , "numbers"
   , "prelude"
   , "psci-support"
```

Comment:
This package is always included by Spago now, so it can be removed from the dependency lists of these various packages. Nothing needs to list it.

Reply:
Alright, I'll remove it then 👍

Reply:
@thomashoneyman it looks like it is still has to be a dependency as of `psc-0.15.2-20220613` since `spago install` fails with:
```
[error] Some of your project files import modules from packages that are not in the direct dependencies of your project.
To fix this error add the following packages to the list of dependencies in your config:
- numbers
You may add these dependencies by running the following command:
spago install numbers
```

Reply:
Sorry, this diff is a little confusing. I commented on the line with “psci-support” and meant to refer to that package.

Reply:
Okay, whoops. Definitely thought you meant `numbers` 😅